MarsRoverTest/MarsRoverTest/Rover.cs

```diff
@@ -20,10 +20,27 @@ public Rover()
             Y = 0;
             Direction = Direction.North;
         }
+        public void MoveForward()
+        {
+            if (Direction == Direction.North)
+            {
+                Y = Y + 1;
+            }
+            else if (Direction == Direction.South)
+            {
+                Y = Y - 1;
+            }
+           else if (Direction == Direction.East)
```

Comment:
Total nitpick here as well, can we get this `else if` lined up correctly? You can Visual Studio do this automatically for you by using ctrl+k followed by ctrl+d (or by going to the menu bar -> Edit -> Advanced -> Format Document)